doc/changelog/02-specification-language/18254-master+def-in-context-instance-in-modtype.rst

```diff
@@ -0,0 +1,8 @@
+- **Changed:**
+  Definitions done using :cmd:`Context` outside a section in a
+  :cmd:`Module Type` do not any more try to declare a class
+  instance. Assumptions bound to a class and declared using
```

Comment:
"bound to a class" sounds wrong, it's more "whose type is a class" no?

Reply:
I was not satisfied either. Your suggestion is ok to me.